packages/backend/src/tokens/jwt/assertions.test.ts

```diff
@@ -105,4 +105,63 @@ export default (QUnit: QUnit) => {
       );
     });
   });
+
+  module('assertHeaderType', () => {
+    test('does not throw error if type is missing', assert => {
+      assert.equal(undefined, assertHeaderType(undefined));
+    });
+
+    test('throws error if type is not JWT', assert => {
+      assert.raises(() => assertHeaderType(''), `Invalid JWT type "". Expected "JWT".`);
+      assert.raises(() => assertHeaderType('Aloha'), `Invalid JWT type "aloha". Expected "JWT".`);
+    });
+  });
+
+  module('assertHeaderAlgorithm', () => {
+    test('does not throw error if type is missing', assert => {
+      assert.equal(undefined, assertHeaderType(undefined));
+    });
+
+    test('throws error if type is not JWT', assert => {
+      assert.raises(() => assertHeaderType(''), `Invalid JWT type "". Expected "JWT".`);
+      assert.raises(() => assertHeaderType('Aloha'), `Invalid JWT type "aloha". Expected "JWT".`);
+      assert.raises(() => assertHeaderType(null), `Invalid JWT type "null". Expected "JWT".`);
+    });
+  });
+
+  module('assertSubClaim', () => {
+    // todo('add tests', assert => {
+    //   assert.true(true);
+    // });
+  });
+
+  module('assertAuthorizedPartiesClaim', () => {
+    // todo('add tests', assert => {
+    //   assert.true(true);
+    // });
+  });
+
+  module('assertIssuerClaim', () => {
+    // todo('add tests', assert => {
+    //   assert.true(true);
+    // });
+  });
+
+  module('assertExpirationClaim', () => {
+    // todo('add tests', assert => {
+    //   assert.true(true);
+    // });
+  });
+
+  module('assertActivationClaim', () => {
+    // todo('add tests', assert => {
+    //   assert.true(true);
+    // });
+  });
+
+  module('assertIssuedAtClaim', () => {
+    // todo('add tests', assert => {
+    //   assert.true(true);
+    // });
+  });
```

Comment:
Can you please write those tests before we get this merged? Thanks!

Reply:
I've updated the tests.